src/libtest/stats.rs

```diff
@@ -907,7 +907,8 @@ mod tests {
 
 #[cfg(test)]
 mod bench {
-    use Bencher;
+    extern crate test;
+    use self::test::Bencher;
```

Comment:
This seems a little odd, but this is the whole "how does test test itself" problem, right? If so, could these tests be moved to an external location like `src/libtest/benches/stats.rs`?

Reply:
Yeah so now libtest is being tested by the compiler's libtest.

The first two are actual benchmarks of the stats `sum` method, but I think all 3 are really just smoke tests and may make most sense as a compiletest.